### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by poudenes** [djtimca/hasatellitetracker#3]
Hi All,

I see many errors in my log:

I use add-on 0.0.4 on HA 2020.12.2

```
2021-01-05 18:35:26 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 119, in _handle_refresh_interval
    await self.async_refresh()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 193, in async_refresh
    update_callback()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 245, in _handle_coordinator_update
    self.async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 296, in async_write_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 320, in _async_write_ha_state
    sstate = self.state
  File "/usr/src/homeassistant/homeassistant/components/binary_sensor/__init__.py", line 158, in state
    return STATE_ON if self.is_on else STATE_OFF
  File "/config/custom_components/satellitetracker/binary_sensor.py", line 116, in is_on
    next_pass = self.coordinator.data["visual_passes"][0]
IndexError: list index out of range
```

**Comment by djtimca** [comment]
I'm assuming you have no upcoming visual passes showing. Will update and put in a new release to check for that condition shortly.

**Comment by djtimca** [comment]
Fixed in 0.0.5
